scala/core/src/partprops/PropMap.scala

```diff
@@ -62,6 +99,12 @@ object PropMap {
     new PropMap(Map[Property, Any]())
   }
 
+  @targetName("PropMapConvenienceConstructor")
```

Comment:
Are you trying to export this to JS?

Reply:
No. `@targetName` is just for disambiguation between different methods that have the same type after erasure.